specificatie/datum.yaml

```diff
@@ -55,12 +59,16 @@ components:
             onbekend:
               type: boolean
               default: true
+            langFormaat:
+              type: string
```

Comment:
Hier zou je het pattern ^onbekend$ of een enumeration met alleen de waarde 'onbekend' toe kunnen voegen.
Nieuwsgierig naar welke van deze 2 de voorkeur heeft.